src/containers/shared/components/Transaction/PaymentChannelCreate/parser.ts

```diff
@@ -0,0 +1,26 @@
+import { convertRippleDate } from '../../../../../rippled/lib/convertRippleDate'
+import formatAmount from '../../../../../rippled/lib/txSummary/formatAmount'
+import { PaymentChannelCreate } from './types'
+
+const findNode = (meta: any, nodeType: string) => {
```

Comment:
This function name feels a bit too generic considering it only applies for `PayChannel` funcitons. Maybe `findPayChannelNode` would be a more accurate name?

Reply:
(Unless it has to be that name to fit a different format? I don't quite remember if this is part of the interface for parsers)

Reply:
I am going to refactor this once these are all merged.  There are a bunch of `findNode`s that are very very close to each other.

Reply:
Sounds good :)